scripts/compare_benchmarks.py

```diff
@@ -57,13 +47,48 @@ def calculate_and_format_p_value(old, new):
         return "(run time too short)"
     elif (len(old_durations) < min_iterations or len(new_durations) < min_iterations):
         is_significant = False
-        return "(not enough runs)"
+        global add_note_for_insufficient_pvalue_runs
+        add_note_for_insufficient_pvalue_runs = True
+        return colored('˅', 'yellow', attrs=['bold'])
```

Comment:
```
       # in case we cannot decide whether the change is significant due to a lack of measurement, a note is added
        # to the table (later added during table printing)
        global add_note_for_insufficient_pvalue_runs
        add_note_for_insufficient_pvalue_runs = True
        return colored('˅', 'yellow', attrs=['bold'])
```

Ok?